librosa/effects.py

```diff
@@ -287,13 +290,15 @@ def pitch_shift(y, sr, n_steps, bins_per_octave=12):
     ...                                          bins_per_octave=24)
     '''
 
-    if bins_per_octave < 1 or not np.issubdtype(type(bins_per_octave), np.integer):
+    if bins_per_octave < 1 or not np.issubdtype(type(bins_per_octave),
```

Comment:
not sure why git diff picks up these two lines? did anything change?

Reply:
looks like OP's text editor automatically wrapped a line here.  NBD.

Reply:
> not sure why git diff picks up these two lines? did anything change?

Oh yeah cuz I made a new line before np.integer to comply with the "80 char per line limit"